src/Components/BacktoTopButton.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import { BiArrowToTop } from 'react-icons/bi';
+import styled from 'styled-components';
+
+export default function BacktoTopButton() {
+  const [backtoTop, setBacktoTop] = useState(false);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > 500) {
+        setBacktoTop(true);
+      } else {
+        setBacktoTop(false);
+      }
+    });
+  }, []);
+
+  const scroleToTop = () => {
```

Comment:
"scrollToTop" maybe? :D 
 You can always change ALL appearances with "cmd+F" or by selecting all similar words with "cmd+shift+L"